### ISSUE TITLE
Godot doesn't play nice with multi-line example out puts

### ISSUE THREAD

**Issue by webstradev** [tetafro/godot#35]
So just wanted to raise another similar issue thought it would be easier to reopen this one.
This fix only works for single line output, but a lot of example tests often have multiple lines and the pattern would be something like this:
```go
// Output:
// true
// false
```

Also, go supports unordered output like this:

```go
// Unordered Output:
// false
// true
```

_Originally posted by @webstradev in https://github.com/tetafro/godot/issues/29#issuecomment-2242218455_

**Comment by tetafro** [comment]
Fixed in [v1.4.17](https://github.com/tetafro/godot/releases/tag/v1.4.17)

**Comment by webstradev** [comment]
@tetafro Thanks for this release, though testing this with the formatting (-w flag) seems to show that both multiline and singleline // Output: still gets a . automatically put to the end of the comment.

**Comment by webstradev** [comment]
Some steps to reproduce

testfile
```go
func ExampleDoSomething() {
	DoSomething()
	// Output:
	// hello
}

func ExampleDoSomethingElse() {
	DoSomethingElse()
	// Output: hello
}

```

.godot.yml
```yaml
scope: all

# List of regexps for excluding particular comment lines from check.
exclude:

# Check periods at the end of sentences.
period: true

# Check that first letter of each sentence is capital.
capital: false
```

Command used to run formatter:
```bash
go run github.com/tetafro/godot/cmd/godot@v1.4.17 -w .
```

In both cases a dot is added by the formatter.

**Comment by webstradev** [comment]
And actually just to add to this, if I run it without -w it still comes up as a linter warning as well.

**Comment by tetafro** [comment]
@webstradev lol, right. Never trust unit tests :)
It is fixed _now_ in [v1.4.18](https://github.com/tetafro/godot/releases/tag/v1.4.18). At least for your example.

**Comment by webstradev** [comment]
Thanks @tetafro Seems to be working on our entire repo (which has a large amount of example tests).

Thanks for the quick fix!

as far as I'm concerned this can be closed
